index.js

```diff
@@ -127,21 +127,20 @@ function animaster() {
 
         heartBeating(element) {
 
-            const timer1 = setInterval(() => {
-                this.scale(element, 500, 1.4)
-            }, 500)
-
-            const timer2 = setInterval(() => {
-                this.scale(element, 500, 1)
-            }, 1000)
+            const duration = 1000;
+            const maxRatio = 1.4;
+            const minRatio = 1;
+            let timer = setInterval(() => {
+                this.scale(element, duration / 2, maxRatio);
+                setTimeout(this.scale, duration / 2, element, duration / 2, minRatio);
```

Comment:
Лучше передавать функцию без параметров для улучшения читаемости
```suggestion
                setTimeout(() => this.scale(element, duration / 2, minRatio), duration / 2);
```